data/map planets.txt

```diff
@@ -3450,6 +3531,10 @@ planet "New Portland"
 	description `New Portland is a cloudy planet orbiting a dim star; the only reason it is not completely frozen is because of the heat trapped by its upper atmosphere. There are only a handful of towns here, and nothing big enough to warrant being called a city. Most of the settlers are subsistence farmers who can only dream about life on other worlds. Many houses here do not even have running water or personal computers.`
 	description `	Some of the farms near the spaceport are planted with different crops forming abstract patterns visible from the air. Such a whimsical extravagance seems particularly odd when you consider that the owners will never be able to afford to see their fields from that angle. But perhaps that is precisely the point.`
 	spaceport `The largest town on New Portland has set aside a landing area for ships, but it is barely what could be called a spaceport. Aside from the refueling station and a few sheds, there are no other buildings, just an open field of packed gravel, with grass and weeds peeking up in places where it has avoided the blast from landing spacecraft. A few farmers have parked wagons here and are trading food and other goods with the visiting ships.`
+	spaceport `	It appears that the spaceport has not been embellished for Remembrance Day. You assume that the main celebrations are taking place in the planet's individual towns.`
```

Comment:
This is borderline telling me what I think. It's probably okay, but you could also do something along the lines of:
```suggestion
	spaceport `	It appears that the spaceport has not been embellished for Remembrance Day. The main celebrations are likely taking place in the planet's individual towns.`
```